build.sbt

```diff
@@ -10,16 +10,31 @@ resolvers ++= Seq(
 
 libraryDependencies ++= Seq(
   "ch.qos.logback"           % "logback-classic"          % "1.1.2",
+  "com.rojoma"              %% "rojoma-json-v3"           % "3.3.0",
+  "com.rojoma"              %% "rojoma-json-v3-jackson"   % "1.0.0"
+    exclude("com.rojoma", "rojoma-json-v3_2.10"),
   "com.rojoma"              %% "simple-arm-v2"            % "2.1.0",
-  "com.socrata"             %% "socrata-http-client"      % "3.3.0",
-  "com.socrata"             %% "socrata-http-jetty"       % "3.3.0",
+  "com.socrata"             %% "socrata-http-client"      % "3.3.0"
+    exclude("com.rojoma", "rojoma-json_2.10")
+    exclude("com.rojoma", "rojoma-json-v3_2.10")
+    exclude("com.rojoma", "rojoma-json-v3-jackson_2.10")
+    exclude("com.rojoma", "simple-arm_2.10")
+    exclude("com.rojoma", "simple-arm-v2_2.10")
+    exclude("com.socrata", "socrata-thirdparty-utils_2.10"),
+  "com.socrata"             %% "socrata-http-jetty"       % "3.3.0"
+    exclude("com.rojoma", "rojoma-json_2.10")
```

Comment:
Think there is some way to exclude everything from com.rojoma, if that's simpler.